.env.development

```diff
@@ -1 +1,2 @@
-NEXT_PUBLIC_BASE_URL='/api'
\ No newline at end of file
+NEXT_PUBLIC_BASE_URL='https://hp-api.fly.dev/api/v1'
```

Comment:
Do we have our branding URL?

Reply:
No. Just waiting a proper url from BE boilerplate and I will replace it later. cc: @hieuphq